model-optimizer/mo_caffe.py

```diff
@@ -3,16 +3,7 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-import sys
-
-from mo.utils.versions_checker import check_python_version
 
 if __name__ == "__main__":
-    ret_code = check_python_version()
-    if ret_code:
-        sys.exit(ret_code)
-
-    from mo.main import main
-    from mo.utils.cli_parser import get_caffe_cli_parser
-
-    sys.exit(main(get_caffe_cli_parser(), 'caffe'))
+    from mo.main import subprocess_main # pylint: disable=no-name-in-module
```

Comment:
pylint disable warning is printed here, but not in other places like mo_kaldi

Reply:
changed.